assets/css/index.css

```diff
@@ -0,0 +1,756 @@
+  color: var(--neutral);
+  background-color: var(--secondary);
+  padding: 0.5em 1.25em;
+  border-radius: 0.25em;
+}
+
+/* =============================End of Header Section============================= */
+
+/**********************************************************************************   */
+
+/* =============================Start of Intro Section============================ */
+.section-intro {
+  display: flex;
+  background-image: url('/assets/img/mobile-intro-bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: auto;
+  padding: 45px 25px;
+}
+
+.headline {
+  font-family: "cocogoose", Verdana, Tahoma, sans-serif;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  width: 100%;
+  text-align: center;
+
+  /* padding: 30px 0; */
+}
+
+.title-txt {
+  color: var(--secondary);
+}
+
+.title-txt p {
+  font-family: "lato", Verdana, Tahoma, sans-serif;
+  font-size: 20px;
+  margin: 0;
+}
+
+.title-txt h1 {
+  font-size: 22px;
+  padding: 2.5px 0 2.5px  5px;
+
+  /* font-weight: lighter; */
+  margin: 0;
+}
+
+.body-txt {
+  font-family: "lato", Verdana, Tahoma, sans-serif;
+  text-align: justify;
+  font-size: 15px;
+  line-height: 1.5rem;
+  color: var(--primary);
+  background: var(--bg_color);
+  padding: 15px 5px;
+}
+
+.location-txt {
+  padding: 30px 0;
+  text-align: center;
+}
+
+.location-txt h2 {
+  font-family: Verdana, Tahoma, sans-serif;
+  font-size: 15px;
+  color: var(--primary);
+  margin: 10px;
+}
+
+.location-txt p {
+  font-family: "lato", Verdana, Tahoma, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  color: var(--primary);
+  margin: 10px;
+}
+
+/* ===============================End of Intro Section============================= */
+
+/**********************************************************************************   */
+
+/* ===========================Start of Programs Section========================== */
+.section-programs {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--primary);
+  background-image: url('/assets/img/pattern_bg.png');
+  background-repeat: repeat;
+
+  /* background-position: center; */
+  width: 100%;
+  min-height: 950px;
+
+  /* padding: 55px 5px; */
+  padding: 75px 10px 25px;
+}
+
+.section-title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 0;
+  margin: 0 10px 75px;
+}
+
+.section-title h2 {
+  font-family: "lato", Verdana, Tahoma, sans-serif;
+  font-size: 24px;
+  text-align: center;
+  color: var(--neutral);
+  margin: 0;
+}
+
+.speakers-title h2 {
+  font-family: "lato", Verdana, Tahoma, sans-serif;
+  font-size: 24px;
+  text-align: center;
+  color: var(--primary);
+  margin: 0;
+}
+
+#speakers-title h2 {
+  color: var(--primary);
+}
+
+.underline-bar {
+  width: 100px;
+  height: 16px;
+  border-bottom: 3px solid #ef513c;
+
+  /* margin: 0 auto 55px; */
```

Comment:
[Optional]
You should remove commented-out code blocks to make our code cleaner. Git helps us to see the old versions of our code, you can easily check the old versions of files with git.